sonatype/src/main/scala/org/typelevel/sbt/TypelevelSonatypePlugin.scala

```diff
@@ -50,30 +37,22 @@ object TypelevelSonatypePlugin extends AutoPlugin {
         "project /",
         "+mimaReportBinaryIssues",
         "+publish",
-        "tlSonatypeBundleReleaseIfRelevant")
+        "sonaRelease")
     }
   )
 
-  @nowarn("cat=deprecation")
   override def buildSettings =
     Seq(
-      tlSonatypeUseLegacyHost := false,
-      autoAPIMappings := true,
-      sonatypeCredentialHost := {
-        Option(System.getenv("SONATYPE_CREDENTIAL_HOST")).filter(_.nonEmpty).getOrElse {
-          if (tlSonatypeUseLegacyHost.value)
-            Sonatype.sonatypeLegacy
-          else
-            Sonatype.sonatype01
-        }
-      }
+      autoAPIMappings := true
     )
 
   override def projectSettings = Seq(
     publishMavenStyle := true, // we want to do this unconditionally, even if publishing a plugin
-    sonatypeProfileName := organization.value,
-    publishTo := sonatypePublishToBundle.value,
-    commands += sonatypeBundleReleaseIfRelevant,
+    publishTo := {
+      val centralSnapshots = "https://central.sonatype.com/repository/maven-snapshots/"
+      if (isSnapshot.value) Some("central-snapshots" at centralSnapshots)
```

Comment:
Sanity check because I haven't looked: does central-snapshots support non-`SNAPSHOT` suffixed versioned? The reason I ask is because, if so, we could conceivably explore shifting publication of non-final versions to that venue, avoiding the version pollution that everyone loves so much.

Not in this PR though, just as a follow up.

Reply:
Not tried, but even trivial bundle uploads take upwards of ten minutes.  There's no more waiting for it to sync to Maven Central after CI passes, but the upload is going to chew a lot of Actions minutes.  That's another reason to try shift builds that don't _need_ to be in Central.